apps/public-docsite-v9/src/Concepts/Upgrade/FromV0/Components/Popup.stories.mdx

```diff
@@ -0,0 +1,128 @@
+import { Meta } from '@storybook/addon-docs';
+
+<Meta title="Concepts/Upgrading/from v0/Components/Popup Upgrade" />
+
+# Popup Upgrade
+
+## Overview:
+
+Before:
+
+```tsx
+import { Popup, Button } from '@fluentui/react-northstar';
+const Component = () => <Popup trigger={<Button>trigger</Button>} content="content" />;
+```
+
+After:
+
+```tsx
+import { Popover, PopoverTrigger, PopoverSurface, Button } from '@fluentui/react-components';
+const Component = () => (
+  // v0 Popup has default position/align above/start, while v9 has default above/center. Therefore adding `positioning` prop here
+  // v0 Popup no arrow by default, while v9 has. Therefore adding `noArrow` prop here
+  <Popover positioning={'above-start'} noArrow>
```

Comment:
`Popover` no longer has a `noArrow` prop, [it was changed to `withArrow`](https://github.com/microsoft/fluentui/pull/23205) to align with `Tooltip`.